core/reporter-api/src/main/java/org/arquillian/reporter/api/model/entry/FileEntry.java

```diff
@@ -1,9 +1,61 @@
 package org.arquillian.reporter.api.model.entry;
 
+import java.io.File;
+import java.nio.file.Path;
+
 /**
  * An {@link Entry} representing a file
  *
  * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
  */
 public class FileEntry implements Entry {
+
+    private String path;
+
+    /**
+     * Build object by setting file path to Path.toString().
+     * @param path file.
+     */
+    public FileEntry(Path path) {
+        this.path = path.toString();
```

Comment:
I think that relative path makes everything more portable. Suppose next scenario, you run build in node1, and you store the reports at `/home/node1/jenkins/workspace/myproject/target/reports`. Then you publish this report to an http server that might be in the same node or not, or you replicate that server, then you have a problem to adapt all the paths. If you do relative then when you publish something, everything still works.